keyboards/splitkb/kyria/keymaps/lms_ace01/keymap.c

```diff
@@ -0,0 +1,852 @@
+                                 _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    ),
+// /*
+//  * Layer template
+//  *
+//  * ,-------------------------------------------.                              ,-------------------------------------------.
+//  * |        |      |      |      |      |      |                              |      |      |      |      |      |        |
+//  * |--------+------+------+------+------+------|                              |------+------+------+------+------+--------|
+//  * |        |      |      |      |      |      |                              |      |      |      |      |      |        |
+//  * |--------+------+------+------+------+------+-------------.  ,-------------+------+------+------+------+------+--------|
+//  * |        |      |      |      |      |      |      |      |  |      |      |      |      |      |      |      |        |
+//  * `----------------------+------+------+------+------+      |  |      +------+------+------+------+----------------------'
+//  *                        |      |      |      |      |      |  |      |      |      |      |      |
+//  *                        |      |      |      |      |      |  |      |      |      |      |      |
+//  *                        `----------------------------------'  `----------------------------------'
+//  */
+//     [_LAYERINDEX] = LAYOUT(
+//       _______, _______, _______, _______, _______, _______,                                     _______, _______, _______, _______, _______, _______,
+//       _______, _______, _______, _______, _______, _______,                                     _______, _______, _______, _______, _______, _______,
+//       _______, _______, _______, _______, _______, _______, _______, XXXXXXX, XXXXXXX, _______, _______, _______, _______, _______, _______, _______,
+//                                  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+//     ),
+};
+// clang-format on
+
+/*
+ * INITIALIZING FUNCTIONS
+ */
+void matrix_init_user(void) {
+#ifdef ENCODER_ENABLE
+    enc_mode = ENC_MODE_CHAR_LINE;
+#endif
+}
+
+/*
+ * LAYER FUNCTIONS
+ */
+layer_state_t layer_state_set_user(layer_state_t state) {
+    // Activate Settings layer when both Symbol and Navigation layers are on
+    return update_tri_layer_state(state, _SYMBOL, _NAVIGATION, _SETTINGS);
+}
+
+/*
+ * MACRO FUNCTIONS
+ */
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    // Get current modifier status (Normal and One Shot)
+    uint8_t mod_state = get_mods();
+    uint8_t osm_state = get_oneshot_mods();
+
+    // Process keycode
+    switch (keycode) {
+        case CK_QUOT:  // Single/Double Quote
+            if (record->event.pressed) {
+                if ((mod_state | osm_state) & MOD_MASK_SHIFT) {
```

Comment:
Key Overrides can probably handle this better.

Reply:
With spanish keyboard, ES_QUOT sends single quote (') on normal mode and question mark (?) on shifted mode. In addition to this, I would like to have another key with default behaviour: single quote (') on normal mode and double quote (") on shifted mode. This should be as simple as use KC_QUOT keycode, but when spanish keyboard activated it does not work in that way. If I override ES_QUOT with ES_2 when shifted, I would lose spanish behaviour. I didn't find another way to accomplish this.

Reply:
Ah, okay, no worries then.